src/02-requirements/README.md

```diff
@@ -10,19 +10,22 @@ the fact that `extern crate` is not necessary in the 2018 edition.
 
 Also, to follow this material you'll need the following hardware:
 
-- A [micro:bit] board.
+- A [micro:bit v2] board, alternatively a [micro:bit v1] board.
```

Comment:
Should we say preferably V1.5? since V1.3 has some differing peripherals.

Reply:
Yeah being more precise with versions can never hurt